test/IDE/complete_call_pattern_heuristics.swift

```diff
@@ -0,0 +1,44 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-ide-test -batch-code-completion -source-filename %s -filecheck %raw-FileCheck -completion-output-dir %t -code-complete-call-pattern-heuristics
```

Comment:
Renamed from `test/IDE/complete_value_expr_call_pattern_heuristics.swift`
This is now a dedicated test file for `-code-complete-call-pattern-heuristics`